webapp/src/pages/ResponseList/components/ResponseListForm.js

```diff
@@ -98,47 +97,56 @@ class ResponseListForm extends Component {
             // Check if anwser should be displayed in table based on state.selected, then extract only the value's
             val.answers.forEach(answer => {
                 // format anwers display based on type
-
+                const question = questions.find(q => answer.question_id === q.question_id);
                 if (answer.type.includes("text")) {
```

Comment:
Can you remove the if statements that check the answer.type from here and just rely solely on the AnswerValue (and add a tooltip to all of them) - the logic here is duplicating what's in AnswerValue anyway and also isn't quite right

Reply:
Noted. I've removed this in  https://github.com/deep-learning-indaba/Baobab/commit/d530416038e6e2c931e0daac9358697106d22572.

>  // format anwers display based on type
☝️ The comment there is why I'd thought it was important to leave the IFs in place.